src/components/manage/Contents/Contents.jsx

```diff
@@ -151,6 +151,10 @@ const messages = defineMessages({
     id: 'Item(s) pasted.',
     defaultMessage: 'Item(s) pasted.',
   },
+  messageWorkflowUpdate: {
+    id: 'Item(s) workflow has been updated.',
```

Comment:
The modal dialog says "Change state", the Contents toolbar says "Change state", we should keep the wording. Editors may not exactly know what "workflow" means.